release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3510,4 +3510,27 @@ $ oc adm release info 4.11.38 --pullspecs
 [id="ocp-4-11-38-upgrading"]
 ==== Updating
 
+To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-11-39"]
+=== RHSA-2023:2014 - {product-title} 4.11.39 bug fix and security update
+
+Issued: 2023-05-02
+
+{product-title} release 4.11.39 is now available. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:2014[RHSA-2023:2014] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:2013[RHBA-2023:2013] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.39 --pullspecs
+----
+
+[id="ocp-4-11-39-bug-fixes"]
+==== Bug fixes
+* Previously, the Ingress Operator published certificates and keys of all Ingress Controllers to the `router-certs` secret in the `openshift-config-managed` namespace. The Authentication Operator is the only operator that uses the `router-certs` secret to access the cluster Ingress domain. Publishing unnecessary certificates and keys can impact the generation of the secret, because the secret exceeds the maximum resource size limit. For the {product-title} {product-version} release, the Ingress Operator publishes certificates and keys for any Ingress Controllers that relate to the cluster Ingress domain. This change ensures that the Ingress Operator can generate a `router-certs` secret that the Authentication Operator can access for OAuth authentication purposes.  (link:https://issues.redhat.com/browse/OCPBUGS-8000[*OCPBUGS-8000*])
```

Comment:
```suggestion
* Previously, the Ingress Operator failed to publish the `router-certs` secret on a cluster with many Ingress Controllers  because of the secret size limit of 1 mebibyte. Consequently, the Authentication Operator that uses the `router-certs` secret to access the cluster Ingress domain might not have the latest certificate to use for OAuth purposes. With this update, the Ingress Operator publishes the default certificate and key for the Ingress Controller, which has the cluster Ingress domain. This update ensures that the Ingress Operator no longer fails to publish the `router-certs` secret and the Authentication Operator can read and use an up-to-date certificate for OAuth authentication. (link:https://issues.redhat.com/browse/OCPBUGS-8000[*OCPBUGS-8000*])
```